soe/node/SOE_VGW_WCS/README.md

```diff
@@ -0,0 +1,24 @@
+
+
+# SOE-VGW-WCS
+This code is developed to illustrate how Service Orchestration Engine (SOE) can be implemented for IBM Voice Gateway (VGW).
+This code connects the VGW to a Watson Conversation Service (WCS), with intermediate actions.  
+
+
+## Usage
+This code is a startup code towards implementing SOE for IBM Voice Gateway in NodeJS.
+
+
+## Developing
+This code does the following actions:
+1) Gets input from IBM Voice Gateway (STT).
+2) Passes on the input to WCS.
+3) Gets response from WCS.
+4) Add TTS customization tags to the response
+5) Add Voice Gateway customization tags to the response.
+6) Trigger mail if intents detected.
+7) Send back output to IBM Voice Gateway (TTS)  
+
+
+### Tools
+
```

Comment:
Would be good if you could provide steps on how to get it up and running?

From the `manifest.yml` looks like it can be deployed on IBM Cloud as a Cloud Foundry app, is that correct? If yes can you point to how to deploy it?

Otherwise it looks good, thanks for the PR

Reply:
Hi Jose,

Thank you for your reply.
We have enhanced the deployment steps in README.md file

Please review.